packages/eslint-plugin/tests/rules/use-lifecycle-interface/cases.ts

```diff
@@ -120,6 +138,20 @@ export const invalid = [
         },
       },
     ],
+    annotatedOutput: `
+        @Injectable()
+        class Test implements DoBootstrap {
+          ngDoBootstrap() {}
+          
+          
+          
+          ngOnInit() {}
+          
+          
+          ngOnDestroy() {}
+          
+        }
+      `,
```

Comment:
This is the test case I was unable to get to work

Reply:
Hey @bulldog98, could you try the following?
```ts
    annotatedOutput: `
        @Injectable()
        class Test implements DoBootstrap {
          ngDoBootstrap() {}
                       

          ngOnInit() {}
                  

          ngOnDestroy() {}
          
        }
      `,
```

Reply:
Yes that does the trick. I updated the PR. Thanks @abaran30